scripts/run_integration_test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Script to locally run the functional integration test.
+# This script assumes that integration tests are already built under ./bazel-bin directory
+# ($ bazel build --config=PLATFORM integration-test/src/...)
+#
+
+declare -a required_files=("./bazel-bin/integration-test/src/python/http_server/http-server" \
```

Comment:
We should define these as constants and reuse here and below, which guarantees that what we check for and what we use stay in sync. e.g.,
```
HTTP_SERVER="./bazel-bin/integration-test/src/python/http_server/http-server"
```